src/logger.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import os
+from pathlib import Path
+
+
+def setup_logger(name: str) -> logging.Logger:
+    """Настройка логгера для модуля"""
+    # Создаем папку для логов
+    log_dir = Path("logs")
```

Comment:
Лучше так:
```suggestion
    log_dir = Path(__file__).parent.joinpath('logs')
```
Тогда директория с логами всегда будет в корне проекта